C-Plus-Plus/other/BrianKenighan'sAlgorithm.cpp

```diff
@@ -0,0 +1,60 @@
+// This is a preferred approach to count no. of set bits of a digit.
+
+//  Brian Kernighan’s Algorithm:
+<<<<<<< HEAD
```

Comment:
You might have not seemed to use Git properly to resolve this?

Reply:
I have followed contributing.md file sir. Don't think that I have any git issues/